blimflix/client/app/core/sidebar.scss

```diff
@@ -0,0 +1,75 @@
+.sidebar-wrapper {
+height: 100%;
+background-color: #222;
+position: fixed;
+left: 0;
+top: 50px;
+padding: 0;
+padding-top: 70px;
+border-right: 2px solid black;
+}
+
+.profile-wrapper {
+height: 50px;
+color: #FFFFFF;
+}
+
+.row {
+    color: #FFFFFF;
+}
+
+.row img {
+    width: 100%;
```

Comment:
also, as comment
in sass and less you can nest classes and properties to make the code more understandable and compact (KISS)